package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A room-scale WebVR musical experience.",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mozvr/puzzle-rain.git"
```

Comment:
FWIW, GitHub is smart and preserves this but just throws a `git` warning that the repo has moved. but, we should update this regardless.